mautic/mautic
# Add thumbnails to file manager

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | staging for features or enhancements / 3.0 for bug fixes 
| Bug fix?                               | no
| New feature?                           | yes
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | Fixes https://github.com/mautic/mautic/issues/9288




#### Description:
This PR added thumbnails to file manger.



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Go to emails, try create new email - set to code mode and open file manager
3. See If thumbnails are loaded after patch this PR
